utils/update.py

```
@@ -0,0 +1,21 @@
+import os
+import random
+def replace(folder, old, new):
+    for root, _, files in os.walk(folder):
+        for file in files:
+            path = os.path.join(root, file)
+            try:
+                content = open(path, 'r').read()
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)